contrib/ontologies.json

```diff
@@ -0,0 +1,1052 @@
+[
+    {
+        "Variable":"days_to_emergence",
+        "Variable Description":"Planting date: 2017-04-20",
```

Comment:
description should be something like 'number of days from planting until 1/2 of plants emerged'